Marlin/Configuration.h

```diff
@@ -1243,6 +1282,9 @@
 // For Z_PROBE_ALLEN_KEY see the Delta example configurations.
 //
 
+//Add probe deploy and stow options to the motion menu.
+//#define PROBE_DEPLOY_STOW_MENU
```

Comment:
When `BLTOUCH` is enabled, the menu items for deploy & stow is automatically enabled. I'd suggest this menu should also be enabled automatically for these new probes.

Reply:
I updated to make the default behaviour to enable the menu items with the option to disable.